tests/lit-tests/loop_unroll_varying.ispc

```diff
@@ -0,0 +1,213 @@
+// Test to check '#pragma unroll' functionality for different loop statements
+//   with varying type induction variables.
+
+// RUN: %{ispc} %s --target=host --nowrap -O0 --emit-llvm-text --no-discard-value-names --nostdlib -o - | FileCheck %s -check-prefixes=CHECK_ALL,CHECKO0
+// RUN: %{ispc} %s --target=host --nowrap -O2 --emit-llvm-text --no-discard-value-names --nostdlib -o - | FileCheck %s -check-prefixes=CHECK_ALL,CHECKO2
+
+// XFAIL: LLVM_15_0+
+
+
+#define NUM 1024
+
+
+// CHECK_ALL-LABEL: define void @foo_for___
+
+// CHECKO0:         for_loop:
```

Comment:
Looks like this will not work in Release build - I think it will not have named labels.

Reply:
That's unfortunate due to the reason [here](https://github.com/ispc/ispc/pull/2373#discussion_r998804382).